base/main.tf

```diff
@@ -1,4 +1,12 @@
-terraform {}
+terraform {
+  cloud {
+    organization = "alex-personal-terraform"
+
+    workspaces {
+      tags = ["aws-infrastructure"]
```

Comment:
is it name or tags @Nihisil 

```suggestion
      name = ["aws-infrastructure"]
```

Reply:
It is tags. We can't use name here, since we have two envs (prod and stage) for this module and tf doesn't allow to use env variable in this section.

Reply:
It is tags, we are configuring this module for two envs: prod and stage, and since we can't use env variable in nmae section, we had to use tags to connect one module with two workspaces.

With this code:
```tf
terraform {
  cloud {
    organization = "alex-personal-terraform"

    workspaces {
      name = "devops-ic-${var.environment}"
    }
  }
}
```

![image](https://user-images.githubusercontent.com/475367/210296722-08c09509-dee4-4026-b7b3-d59710fd8a07.png)

Reply:
Ah I see @Nihisil

Reply:
btw for ewa payroll app `name = "devops-ic-${var.environment}"` is working fine, not sure why. Maybe it was allowed in previous tf versions.

https://github.com/nimblehq/ewa-payroll-infra/blob/develop/base/main.tf#L6